app/javascript/groceries/components/check_in_items_list.vue

```diff
@@ -14,9 +14,10 @@ section(v-if="items.length > 0")
         :id='`trip-checkin-item-${item.id}`'
       )
       label.ml-2(:for='`trip-checkin-item-${item.id}`')
-        span.mr-1(:class='{ "text-gray-500": item.skipped }')
+        span(:class='{ "text-gray-500": item.skipped }')
           span {{item.name}}
           span(v-if='item.needed > 1') {{' '}} ({{item.needed}})
+        span {{ ' ' }}
```

Comment:
I'm adding this space in place of the `mr-1`. I don't love this solution because it means that the only amount of spacing we can have here is the width of a space. In this case, that's just about what we want, but I don't like the feeling that it might not have been quite what we want, and I would struggle to come up with a good solution in such a case. The problem is that we want the spacer to be of width X, _unless_ the "Skip" button breaks to begin its own new line, in which case we _don't_ want the spacer to take up with at the beginning of that line, effectively indenting the "Skip" button.